package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "github workflows",
+  "description": "JupiterOne Organization-Wide GitHub Workflows.",
+  "repository": "https://github.com/JupiterOne/.github",
+  "version": "1.0.0",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "clean": "npx rimraf repo",
+    "test": "npm run clean && vitest --no-threads",
+    "typecheck": "tsc -b",
+    "lint": "eslint . --ext .ts",
+    "validate": "npm run lint && npm run typecheck && npm run test"
+  },
+  "config": {
+    "migration": 1
```

Comment:
Why add a migration number here?

Reply:
This is actually leveraged by the tests, you can see it [being copied over here](https://github.com/JupiterOne/.github/blob/7ae7f35e9983c31f9bd0f0c3e28d6402525e2902/tests/utils/setup.ts#L28) in the virtual repo that gets creatd.